### PR TITLE
Integrate Collateral Functionality from MARKET.js

### PR DESCRIPTION
**Pull request by travisdmathis**
##### Description

Implements collateral functionality from MARKET.js

##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)

UI, UX, Tests

##### Refers/Fixes

Fixes: #210

### CODE REVIEW

File: src/util/utils.js [hunk 1]
```diff
@@ -338,7 +345,10 @@ export const getCollateralTokenAddress = (network, quoteAsset) => {
     switch (quoteAsset) {
       case 'ETH':
         return '0x01b8de20c76ed06c7e93068a45951c26f70be3db';
+      case 'WETH':
```
[the remark this answers is not in the document]
Comment (travisdmathis) [reply src/util/utils.js]: This function is only used in tests atm.

File: package.json [hunk 2]
```diff
@@ -45,11 +45,14 @@
     "css-loader": "0.28.7",
     "dotenv": "4.0.0",
     "dotenv-expand": "4.0.1",
+    "ethereum-input-data-decoder": "0.0.12",
     "extract-text-webpack-plugin": "3.0.2",
     "file-loader": "1.1.5",
     "fs-extra": "3.0.1",
     "html-webpack-plugin": "2.29.0",
+    "human-standard-token-abi": "^1.0.2",
```
[the remark this answers is not in the document]
Comment (travisdmathis) [reply package.json]: we actually are still using this abi as well when we're pulling back information about token balances for WETH and such.

File: src/components/SimExchange/WalletComponents/HeaderMenu.js [hunk 3]
```diff
@@ -35,30 +54,175 @@ class HeaderMenu extends Component {
...
+    const { simExchange, web3 } = this.props;
+    const { amount } = this.state;
+    const txParams = {
+      from: web3.web3Instance.eth.coinbase
+    };
+
+    marketjs
+      .withdrawCollateralAsync(
+        simExchange.contract.MARKET_COLLATERAL_POOL_ADDRESS,
+        web3.web3Instance.toBigNumber(amount.number * 1000000000000000000),
```
[the remark this answers is not in the document]
Comment (travisdmathis) [reply src/components/SimExchange/WalletComponents/HeaderMenu.js]: :+1:

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (pelsasser) [reply package.json]: Got it, we should be able to pull the ERC20 abi from the `market-solidity` npm package - which I am pretty sure we are using in some other places.

**Pull request merged**